### DISCUSSION TITLE
Error when downloading controls on local ksserver build

### DISCUSSION THREAD

**Discussion by testwill** [kubescape/kubescape#1228] [General]
```
"msg":"failed to get policies from github release, loading policies from cache","error":"error getting: ControlID_RuleName.csv from: 'https://github.com/kubescape/regolibrary/releases/latest/download' ,error: Get \"https://github.com/kubescape/regolibrary/releases/latest/download/ControlID_RuleName\": unexpected EOF"
```
Why do I report this error when I compile and package the ksserver image by myself, .kubescape is copied from your official image

**Comment by testwill** [comment]
bug fix of cannot read local artifacts   ???

**Comment by craigbox** [comment]
I can hit `https://github.com/kubescape/regolibrary/releases/latest/download/ControlID_RuleName` from my browser, so I can only guess it might be GitHub rate limiting you, or something similar?

**Reply by testwill** [reply]
Maybe
